Dockerfile

```diff
@@ -25,9 +25,7 @@ RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
 COPY --from=builder /app/dist /action-release/dist/
 RUN chmod +x /action-release/dist/index.js
 
-# move the sentry-cli binary to where the entrypoint expects it
-RUN mv /action-release/node_modules/@sentry/cli/sentry-cli /action-release/sentry-cli
-RUN chmod +x /action-release/sentry-cli
+RUN printf '[safe]\n    directory = *\n' > /etc/gitconfig
```

Comment:
Not sure how hacky this is or if it will cause problems on other binaries since each OS can have different implementations

Reply:
it's in a docker container so the os is fixed

Reply:
@asottile-sentry Oh yea duh...brain is slow after the holidays